Assets/scss/components/quesionnaire.scss

```diff
@@ -0,0 +1,255 @@
+.cmp-questionnaire {
+  max-width: toRem(1175); 
+  margin: auto;
+
+  @include max-screen(sm) {
+    max-width: toRem(767);
+    margin: auto;
+   }
+
+   @include breakpoint-between(sm,md) {
+    max-width: toRem(1199);
+    margin: auto;
+   }
+
+  &__container {
+    display: flex;
+    background-color: $grey94;
+    height: toRem(590);
+
+    @include max-screen(sm) {
+      display: flex;
+      flex-direction: column;
+      height: 164vh;
```

Comment:
@chintanpatelrightpoint  Please don't add fix height of entire component we don't know the height of form, let it take the height of form.
Cc:- @jvaishnavrp  @msharmaRP

Reply:
Agreed, images should sometimes have specific heights, but for text content, you almost always want it to flow however long is needed.

You can use a `min-height` to set a starting height, but will grow if needed

Reply:
Thanks for the Feedback this has been addressed